js/main.js

```diff
@@ -0,0 +1,8 @@
+function displayMenu() {
+  document.querySelector('#menu-button').classList.toggle('hidden');
+  document.querySelector('#wide-menu').classList.toggle('hidden');
+}
+
+document.querySelector('#menu-button').addEventListener('click', displayMenu);
+document.querySelector('#wide-menu > div > button').addEventListener('click', displayMenu);
+document.querySelector('#wide-menu > div > a').addEventListener('click', displayMenu);
```

Comment:
- [ ] Kindly add the following functionality: When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears 💯 right now only the `Portfolio` option closes the menu.